doc/sphinx/user/install/docker-container/running-models.md

```diff
@@ -80,3 +80,12 @@ recommend the following workflow:
 
 You are all set. Repeat steps 1-4 of this process as necessary when updating
 your model parameters.
+
+:::{admonition} TODO
+:class: error
+Fix referenced link to Sectio 4.4 Debug or Optimized 
+:::
+
+[^footnote1]: Note that it is possible to mount a directory as writeable into the container. However, this is often associated with file
+permission conflicts between the host system and the container. Therefore, we recommend this slightly more cumbersome, but
+also more reliable workflow.
```

Comment:
Is it not possible to have the footnote text right where the footnote symbol is? It is hard to maintain things if pieces of related text end up in different places in the document.